runtime/src/error.c

```diff
@@ -90,6 +95,51 @@ void chpl_error_explicit(const char *message, int32_t lineno,
   else
     fprintf(stderr, "error: %s", message);
   fprintf(stderr, "\n");
+
+  //This is just a prototype!
+  //Using libunwind directly, should be wrapped
+
+  //Stack unwind
+  #define chpl_stack_frame_t unw_cursor_t
+
+  unw_cursor_t cursor; unw_context_t uc;
+  chpl_stack_frame_t cursor_stack[32];
+  size_t stack_size = 0;
+
+  unw_getcontext(&uc);
+  unw_init_local(&cursor, &uc);
+
+  while (unw_step(&cursor) > 0) {
+    cursor_stack[stack_size++] = cursor;
+  }
```

Comment:
Yep, the prototype overflows if the trace has more than 32 frames. It was just a quick and dirty method for avoiding allocate the space for our trace (maybe based on a stacklet ?), but obviously the final version will need to do that.